### ISSUE TITLE
Customize slide numbers

### ISSUE THREAD

**Issue by Phyks** [hakimel/reveal.js#965]
Hi,

What about offering a way (via a config option for instance) to customize the slide numbers : the way they are displayed (indexh / indexv for instance, instead of indexh - indexv) and to change the first displayed number, to put it equal to 1, which is usually more intuitive for instance ?

Thanks

**Comment by hakimel** [comment]
The slide number format is now configurable using variables. For example, to display "indexh / indexv" you would use `Reveal.configure({ slideNumber: 'h / v' })'.

The default behavior has been changed to showing a flattened index, rather than that "indexh - indexv" split. 

More info: https://github.com/hakimel/reveal.js/tree/dev#slide-number

**Comment by maxmeyer** [comment]
Wonderful! :-)

**Comment by hakimel** [comment]
I just revised how this works. The variable substitution approach I first implemented wasn't great since it meant it always showed the vertical slide number even if there were no vertical slides. Instead we now support a number of preset formats which are listed here: https://github.com/hakimel/reveal.js#slide-number

The slide numbers are now wrapped in markup so that you have more control over the style. This also allows you to substitute the delimiter using CSS (:before/:after pseudo elements).

Slide numbers work in PDF exports now too!
